recipes/epa-ng/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: epa-ng
+  version: "0.3.3"
+
+build:
+  number: 1
+
+source:
+  url: https://github.com/Pbdas/epa-ng/archive/v0.3.3.tar.gz
+  md5: 881d024877f3cfa632d344373e119f9d
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - libtool
+    - flex
+    - bison
+    - m4
+    - cmake==3.11
+    - automake
+    - autoconf
+  run:
```

Comment:
zlib should be a host dependency. The run dependency will be injected automatically.

Reply:
The build/host/run is a little confusing. It's for cross compiling. Everything that is build machinery, so compiler and build tools go into `build`. Everything that is dependent on the build target, so libraries you need to link against, goes into `host`. Things that will be needed at run time go in to `run`. Some packages have what conda calls `run exports`. Those will add things to your `run` automatically. Zlib is one of them. The `run: - zlib` is therefore redundant.

Reply:
Yes I was confused by this; however, without adding zlib as a run requirement this error is causes: ```SERR epa-ng: error while loading shared libraries: libz.so.1: cannot open shared object file: No such file or directory```, which you can see here: https://circleci.com/gh/bioconda/bioconda-recipes/38521?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification

Reply:
It needs to be in `host`.

Reply:
I added that - should work.